scripts/gas-refund-program/persistance/db-persistance.ts

```diff
@@ -47,6 +49,42 @@ export async function fetchVeryLastTimestampProcessed({
   return lastTimestamp;
 }
 
+export async function fetchTotalRefundedPSP(): Promise<BigNumber> {
+  const totalPSPRefunded = await GasRefundParticipation.sum<
+    string,
+    GasRefundParticipation
+  >('refundedAmountPSP');
+
+  return new BigNumber(totalPSPRefunded);
+}
+
+export async function fetchTotalRefundedAmountUSDByAddress(): Promise<{
+  [address: string]: BigNumber;
+}> {
+  const totalRefundedAmountUSDAllAddresses =
+    (await GasRefundParticipation.findAll({
+      attributes: [
+        'address',
+        [
+          Sequelize.fn('SUM', Sequelize.col('refundedAmountUSD')),
+          'totalRefundedAmountUSD',
+        ],
+      ],
+      group: 'address',
+    })) as unknown as { address: string; totalRefundedAmountUSD: string }[];
```

Comment:
They didn't account for arbitrary transforms with `fn` :disappointed:

Reply:
just re-checked, apparently no. That would be a funny type twist though ^^